ghcup-prereleases-0.0.8.yaml

```diff
@@ -2141,13 +2141,163 @@ ghcupDownloads:
+              dlOutput: ghc-9.12.0.20241031-x86_64-linux-deb10.tar.xz
+              dlSubdir: ghc-9.12.0.20241031-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241031/ghc-9.12.0.20241031-x86_64-deb10-linux.tar.xz
+            ( >= 11 && < 12 ): &ghc9121alpha2-x86_64-deb11
+              dlHash: 94f786397f213188c0e26fb6c52a0487d7bdf42f3a1274c2a63b07603abe6418
+              dlOutput: ghc-9.12.0.20241031-x86_64-linux-deb11.tar.xz
+              dlSubdir: ghc-9.12.0.20241031-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241031/ghc-9.12.0.20241031-x86_64-deb11-linux.tar.xz
+            < 10: &ghc9121alpha2-x86_64-deb9
+              dlHash: 44502a6cefcdf66ad1a6d409536ae62ae110a3e088766c865652a0fd34d78a79
+              dlOutput: ghc-9.12.0.20241031-x86_64-linux-deb9.tar.xz
+              dlSubdir: ghc-9.12.0.20241031-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241031/ghc-9.12.0.20241031-x86_64-deb9-linux.tar.xz
+            '>= 12':
+              dlHash: d8ebf6666e00fdafcb1acc5eac39e99fbea4e06b15c23bd33e4f94481b147005
+              dlOutput: ghc-9.12.0.20241031-x86_64-linux-deb12.tar.xz
+              dlSubdir: ghc-9.12.0.20241031-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241031/ghc-9.12.0.20241031-x86_64-deb12-linux.tar.xz
+            unknown_versioning: *ghc9121alpha2-x86_64-deb11
+          Linux_Fedora:
+            '>= 33':
+              dlHash: 9e25d7dc3f332aa6ba45aec535ccde843b46b272407e3828f1216959e6e0e586
+              dlOutput: ghc-9.12.0.20241031-x86_64-linux-fedora33.tar.xz
+              dlSubdir: ghc-9.12.0.20241031-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241031/ghc-9.12.0.20241031-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc9121alpha2-x86_64-centos7
+          Linux_Mint:
+            < 20: &ghc9121alpha2-x86_64-ubuntu18_04
+              dlHash: 2333df8b2f4363ec8a78c92d0f0939fdbea64637eb528f32d7581bfc1a0748a3
+              dlOutput: ghc-9.12.0.20241031-x86_64-linux-ubuntu18_04.tar.xz
+              dlSubdir: ghc-9.12.0.20241031-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241031/ghc-9.12.0.20241031-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc9121alpha2-x86_64-ubuntu22_04
+              dlHash: a0dc0c19509a738acfe3b75d284bda1b2e4a07e43b36a46f9f305985372028eb
+              dlOutput: ghc-9.12.0.20241031-x86_64-linux-ubuntu20_04.tar.xz
+              dlSubdir: ghc-9.12.0.20241031-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241031/ghc-9.12.0.20241031-x86_64-ubuntu20_04-linux.tar.xz
+            unknown_versioning: *ghc9121alpha2-x86_64-ubuntu22_04
+          Linux_RedHat:
+            unknown_versioning: *ghc9121alpha2-x86_64-centos7
```

Comment:
It fails for RedHat 9, which apparently doesn't have libtinfo.so.5 anymore: https://github.com/haskell/ghcup-metadata/actions/runs/11684445548/job/32535716297#step:5:15963

Reply:
You can check here about binary compatibility: https://gitlab.haskell.org/ghc/ghc/-/wikis/platforms/linux

It appears that "Ubuntu 20.04 (focal)" might work.

You can test it in the docker image `registry.access.redhat.com/ubi9/ubi` locally.

Reply:
Additionally... GHC HQ could consider building Rocky9 bindists (which are binary compatible with RedHat9).

Reply:
it looks like ghc-9.12.0.20241031-x86_64-fedora33-linux.tar.xz might actually work better

Reply:
I've changed it to use fedora33 for redhat.